appinfo/routes.php

```diff
@@ -33,6 +33,7 @@
 		['name' => 'api1#createShare',	'url' => '/api/1/shares', 'verb' => 'POST'],
```

Comment:
@blizzz So, now I can create a share. But how do I get all the shares associated with a particular context? Do we need an endpoint like this one, but for contexts?

Also, since a context has a `sharing` property, I would have expected that all shares would be listed here. But even after adding shares using `axios.post(generateUrl('/apps/tables/share'), share)`, the `sharing` array is still empty. 


```
context = {
	'id'
	'name'
	'iconName'
	'description'
	'owner'
	'ownerType'
	'sharing'
	'nodes' 
	'pages' 
}
```

Reply:
Thanks for pointing this out. The reason for an empty sharing value was that the user initiated shares were not queried. Only those where the current  users is a receiver either directly, or by group. 

I just pushed a commit that would fetch share information when the user is also the sender.

For only Context owners can share the context, this should actually changed to fetch all shares when the acting user is the owner. That's a follow up then.